cmd/build/helmify/static/README.md

```diff
@@ -1,9 +1,46 @@
 # Gatekeeper Helm Chart
 
+## Get Repo Info
+
+```console
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+# Helm install with gatekeeper-system namespace already created
+$ helm install -n gatekeeper-system [RELEASE_NAME] gatekeeper/gatekeeper
+
+# Helm install and create namespace
+$ helm install -n gatekeeper-system [RELEASE_NAME] gatekeeper/gatekeeper --create-namespace --set postInstall.labelNamespace.enabled=true
```

Comment:
@ritazh @maxsmythe are we okay with this behavior change? this should be documented in release notes.

Reply:
What is the behavior change? the `--create-namespace` flag?

TBH, I don't think of Helm as part of Gatekeeper's backwards compatibility contract b/c it's an installation method, not an interface, though reasonable people can disagree here. More importantly, it may not be possible to maintain backwards compatibility as it seems like some if these changes are forced by upstream changes to Helm itself?

With that in mind, I would do whatever the community standard is for Helm charts when they need to change.

Reply:
This postInstall hook is definitely not obvious. +1 on adding it to release notes and all the docs.
Can `postInstall.labelNamespace.enabled` be default to true so users don't need to do this every time?

Reply:
The behaviour change here is that the Helm chart no longer creates and manages the Namespace. Helm will create it if you ask it by setting the `--create-namespace` flag, but it's not managed by the chart itself. For example if you uninstall the chart the Namespace will remain. 

The challenge with this chart is that we want the `admission.gatekeeper.sh/ignore` label on the Namespace Gatekeeper is installed in, which is what the postInstall hook stuff does.

As an alternative to this could we include a `Config` in the chart and set a reasonable default to the Namespace, and allow callers to override it? Based on the docs it seems like this is another alternative to the 
`--exempt-namespace` flag.  Also happy to set `postInstall.labelNamespace.enabled` to `true` by default, and remove the `post-upgrade` hook so that it only happens on first time installs. Because that labeling only has to happen one time I would think.

Reply:
> happy to set postInstall.labelNamespace.enabled to true by default, and remove the post-upgrade hook so that it only happens on first time installs. Because that labeling only has to happen one time I would think.

+1